resteasy/src/main/java/com/cerner/beadledom/resteasy/ResteasyContextListener.java

```diff
@@ -76,4 +70,21 @@ protected void withInjector(Injector injector) {
    * Returns the list of Guice modules to be used with Resteasy.
    */
   protected abstract List<? extends Module> getModules(ServletContext context);
+
+  /**
+   * Processes module bindings in the Guice injector.
+   */
+  @VisibleForTesting
+  public void processInjector(ServletContext context, Injector injector) {
```

Comment:
I'm not convinced the infinite loop could happen at all or that the while loop is even needed at all. Since the context listener creates the injector is there even any possible way for the injector to have a parent injector?